nvflare/private/fed/server/job_cmds.py

```diff
@@ -193,8 +193,8 @@ def _send_summary_list(self, conn: Connection, jobs: List[Job]):
             table.add_row(
                 [
                     job.meta.get("job_id"),
-                    job.meta.get("name"),
-                    job.meta.get("study_name"),
+                    job.meta.get("name") or "",
+                    job.meta.get("study_name") or "",
```

Comment:
Maybe job.meta.get("study_name", "") is better.

Reply:
Definitely better. Updated.